docs/extensions/get-started/generator.md

```diff
@@ -0,0 +1,65 @@
+# New Extension Project with Generator
+
+The [Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) scaffolds a project ready for development. Install Yeoman and Lens Extension Generator with:
+
+```bash
+npm install -g yo generator-lens-ext
+```
+
+Run the generator and fill out a few fields for a TypeScript project:
+
+```bash
+yo lens-ext
+# ? What type of extension do you want to create? New Extension (TypeScript)
+# ? What's the name of your extension? my-first-lens-ext
```

Comment:
At least when I run the generator the suggested name is "my-ne-lens-extension"

Reply:
Thanks, fixed a typo, sync the default to generator repo https://github.com/lensapp/generator-lens-ext/pull/8